arches/app/views/graph.py

```diff
@@ -210,23 +225,15 @@ def get(self, request, graphid):
         if self.graph.ontology is not None:
```

Comment:
Clicking on an existing graph in AFS throws on this line because self.graph is None (in turn, because self.editable_future_graph is None, earlier). I ran the `manage.py graph publish --update` command.

Reply:
This is a good catch. `graph publish --update` is really only useful in germs of adding or updating languages. There wasn't a great method to create editable_future_graphs in bulk for a project upgrade. So I added the management command `create_editable_future_graphs` to accommodate